Nero/plugins/Nero.cc

```diff
@@ -190,7 +222,7 @@ Nero::Nero(const edm::ParameterSet& iConfig)
     phos -> mMinPt = iConfig.getParameter<double>("minPhoPt");
     phos -> mMaxIso = iConfig.getParameter<double>("maxPhoIso");
     phos -> mMinNpho = iConfig.getParameter<int>("minPhoN");
-    phos -> mMaxEta = iConfig.getParameter<double>("maxPhoEta");
+    phos -> mMaxEta = iConfig.getParameter<double>("minPhoEta");
```

Comment:
come on ... at least in the place I fixed it ... in any case we need to change all the minEta -> maxEta everywhere in the code. ( eg, don't worry now)